app/controllers/api/v1/invoices/random_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Invoices::RandomController < ApplicationController
+
+	def show
+		render json: Invoice.random
+	end
```

Comment:
This is weird that Github shows the formatting this way. it is NOT like this in my local machine's code.

@sallymacnicholas @tmikeschu